docs/articles_en/get-started/install-openvino/install-openvino-pip.rst

```diff
@@ -84,7 +84,7 @@ Use the following command:
 Step 4. Install the Package
 +++++++++++++++++++++++++++
 
-Use the following command to install OpenVINO Base or OpenVINO GenAI package:
+Use the following command to install either the base or GenAI OpenVINO package:
```

Comment:
what base means? May be it is better to say just "OpenVINO"?

Reply:
That is still problematic due to OpenVINO being the name of the toolkit. These changes are a short-term attempt to make the distinction a bit more pronounced. We should have a better wording for it soon.